### PR TITLE
Closes Define impact level #115

### PR DESCRIPTION
**Pull request by martiL**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -92,6 +92,24 @@ function isHTMLString (html) {
...
 
+/**
+ * Filters all violations by user impact
+ * @param {object} violations result of the accessibilty check by axe
+ * @param {array} impactLevels defines which impact level should be considered (e.g ['critical'])
+ * The level of impact can be "minor", "moderate", "serious", or "critical".
+ * @returns {object} violations filtered by impact level
+ */
+function filterViolations (violations, impactLevels) {
+  if (typeof violations === 'undefined') {
```
Comment (nickcolley) [inline index.js:103]: Could we remove this and move the filter below line 126 so avoid doing this twice?

**Review by nickcolley — APPROVED** [review]
Sorry it's taken so long to review this, it looks great, I have one minor comment if you have time to address otherwise I can get this merged and published.

**Comment by nickcolley** [comment]
I'm going to sort this and get it published anyway.

File: index.js (hunk 1 above)
Comment (martiL) [reply index.js:103]: @nickcolley Sorry for the late reply. I completely forgot about this PR. I am very happy that it has been merged, and I see that you have made the change mentioned. thx!

**Pull request merged**